<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>article_view_dialog</class>
 <widget class="QDialog" name="article_view_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#article_view_dialog{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 5, 68, 255), stop:1 rgba(10, 55, 255, 255));}</string>
  </property>
  <widget class="QTextEdit" name="article_text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>511</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Times New Roman&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pdf_export">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="txt_export">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>TXT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filename_input">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 153, 153);
font: 12pt &quot;Arial&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="translate_button">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>770</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>translate</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>770</y>
     <width>73</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Czech</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Dutch</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>English</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>French</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>German</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Italian</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Polish</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Portuguese</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Spanish</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
